common/p_horde.cpp

```diff
@@ -202,6 +203,8 @@ class HordeState
 	int m_nextSpawn;
 	int m_nextPowerup;
 	corpseCollector_t m_corpses;
+	std::map<mobjtype_t, int> m_monsterCounts;
+	std::map<mobjtype_t, int> m_bossCounts;
```

Comment:
I appreciate you trying to make this C++98 compatible, but I would much prefer if we used `OHashTable` instead of a `std::map`.

Reply:
Okay, changed it. Still figuring out both C++ and Odamex.